src/controllers/post.js

```diff
@@ -1,39 +1,51 @@
 import dbClient from '../utils/dbClient.js'
-
 import { sendDataResponse } from '../utils/responses.js'
+import Joi from 'joi'
 
 export const create = async (req, res) => {
-  const { content } = req.body
-  if (!content) {
-    return sendDataResponse(res, 400, { content: 'Must provide content' })
+  const schema = Joi.object({
+    content: Joi.string().min(3).required()
```

Comment:
Should we have `.max(150)` here too?

Reply:
it was but u said on comments only! but I can modify it if u want to

Reply:
Ha did I? Well this is in the same area, so it could be OK to do it there - you did add the min(3) so I think it makes sense to add the max too in this case.